packages/unmock-core/src/runner/index.ts

```diff
@@ -0,0 +1,66 @@
+import chalk from "chalk";
+import { seedHack } from "../generator";
+export interface IRunnerOptions {
+  maxLoop: number;
+}
+
+const defaultRunnerOptions: IRunnerOptions = {
+  maxLoop: 20,
+};
+
+const errorHandler = (
+  allFailed: boolean,
+  errors: Error[],
+  intermediaryErrors: Array<(string | { message: string})>,
+  cb?: jest.DoneCallback) => {
```

Comment:
TODO: This references `jest` types but the Jest types are not includes as `dependencies` of `unmock-core` so the build is not "self-contained" (also see https://github.com/unmock/unmock-js/pull/201).

Reply:
Yeah, we probably want to do some tests with mocha and also look into peer dependencies.